src/serac/numerics/mesh_utils.cpp

```diff
@@ -167,6 +167,96 @@ std::shared_ptr<mfem::ParMesh> buildCuboidMesh(int elements_in_x, int elements_i
   return std::make_shared<mfem::ParMesh>(comm, mesh);
 }
 
+std::shared_ptr<mfem::ParMesh> buildCylinderMesh(int radial_refinement, int elements_lengthwise, double radius,
+                                                 double height, const MPI_Comm comm)
+{
+  static constexpr int dim                   = 2;
+  static constexpr int num_vertices          = 17;
+  static constexpr int num_elems             = 12;
+  static constexpr int num_boundary_elements = 8;
+
+  // a == 1.0 produces a mesh of a cylindrical "core" surrounded by
+  // a cylindrical "shell", but makes some of the elements in the "core" section
+  // nearly degenerate
+  //
+  // a > 1 makes the "core" section no longer a cylinder, but its elements
+  // are no long nearly degenerate
```

Comment:
```suggestion
  // are no longer nearly degenerate
```

Reply:
I didn't know  you could suggest changes like this in github, that's a great feature!